recipes/elyra-code-viewer-extension/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "elyra-code-viewer-extension" %}
+{% set version = "3.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/elyra-code-viewer-extension-{{ version }}.tar.gz
+  sha256: a3facdb17166334bcb2788b8f02d88cf43d4ce39966cda40b325d6ef234ef518
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - jupyter-packaging
+    - pip
+    - python >=3.7
+  run:
+    - elyra-server ==3.7.0
+    - python >=3.7
+
+test:
+  commands:
+    - exit 0
```

Comment:
We need some kind of test to validate that the package works as needed. Can you do like this [other extension](https://github.com/conda-forge/jupyterlab-unfold-feedstock/blob/30f98b549dbe016584ebc9eb5ba5afcde2d95861/recipe/meta.yaml#L28-L31) does?

Reply:
Is there any reason you didn't also copy the Windows test from the example I linked to?

Reply:
Re-added the test in the interest of getting the past CI. It was removed because the project itself doesnt explicitly support windows, just osx and unix environments (although we're working on it) =)
Is there an way to exclude windows based testing from CI or is it a requirement as part of the conda community?

Reply:
Well, it's fine if it doesn't support. Right now, though, as a noarch package it gets built on Linux and is installable on all the platforms. (I'll note that `elyra` itself is the same way.) Usually the way to keep it off Windows is to have a dependency that doesn't support it. :wink:

If you want to make sure this can't be installed on Windows, then I *think* you can depend on the `__unix` virtual package (though that's only a thing for newish versions of conda.